src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -2630,40 +2467,26 @@ baddate:
         End Function
 
         Friend Function TryTokenAsContextualKeyword(t As SyntaxToken, ByRef k As KeywordSyntax) As Boolean
-            If t Is Nothing Then
-                Return False
-            End If
-
-            If t.Kind = SyntaxKind.IdentifierToken Then
-                Return TryIdentifierAsContextualKeyword(DirectCast(t, IdentifierTokenSyntax), k)
-            End If
-
+            If t Is Nothing Then Return False
```

Comment:
If I might weigh in, we've also pretty consistently avoided single line "Ifs" in VB and "ifs" without "{ ... }" in C# (throughout the codebase).

I personally really dislike trying to read VB single line Ifs unless they are very short (like "If x Then Return False").  Two reasons:
1.  Statement Ifs are just a hold over from BASIC, and the eye naturally searches for the "End ..." in a language where block ifs are the norm.  Not having one creates dissonance (especially if I cannot see the end of the enclosing scope).
2.  It's very easy for the condition or the statement to run off the screen, decreasing readability.